### DISCUSSION TITLE
PrimeTweenManager always appears under DoNotDestroyOnLoad

### DISCUSSION THREAD

**Discussion by zemrearslan** [KyryloKuzyk/PrimeTween#130] [General] [answered]
Hello @KyryloKuzyk1 Even though I am not using any animations from PrimeTweenManager in a scene, PrimeTweenManager always appears under DoNotDestroyOnload when the game starts. However, DoTween's manager is not generated, when I don't use anything particular from that package. Is it supposed to be like this? Thanks for the answers!

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey! Yes, this is an intended behavior. The idea is that PrimeTween allocates memory once at the start of your game to prevent any hiccups in the future once the game decides to animate something. 

There is no way to prevent PrimeTweenManager from spawning for simplicity reasons. But if you want to reduce PrimeTween's memory footprint to the minimum when your game starts, you can call `PrimeTweenConfig.SetTweensCapacity(0)` at the start of your game.

**Reply by zemrearslan** [reply]
Thank you for the answer!
